### PR TITLE
Clean up tear down test

### PR DESCRIPTION
**Pull request by itkg-alavieille**
https://github.com/open-orchestra/open-orchestra-libs/pull/38
https://github.com/open-orchestra/open-orchestra-media-file-bundle/pull/6
https://github.com/open-orchestra/open-orchestra-base-bundle/pull/66
https://github.com/open-orchestra/open-orchestra-mongo-libs/pull/11
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1340
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/116
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/154
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/83
https://github.com/open-orchestra/open-orchestra-user-bundle/pull/86
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/504
https://github.com/open-orchestra/open-orchestra-display-bundle/pull/191
https://github.com/open-orchestra/open-orchestra-bbcode-bundle/pull/5
https://github.com/open-orchestra/open-orchestra-base-api-mongo-model-bundle/pull/13
https://github.com/open-orchestra/open-orchestra-base-api-bundle/pull/73

### CODE REVIEW

File: BaseBundle/Tests/AbstractTest/AbstractBaseTestCase.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+namespace OpenOrchestra\BaseBundle\Tests\AbstractTest;
+
+/**
+ * Class AbstractBaseTestCase
+ */
+class AbstractBaseTestCase extends \PHPUnit_Framework_TestCase
+{
+    use TestCleanUpTearDownTrait;
+}
```
Comment (nicolasThal) [inline BaseBundle/Tests/AbstractTest/AbstractBaseTestCase.php]: eof

File: BaseBundle/Tests/AbstractTest/AbstractKernelTestCase.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+/**
+ * Class AbstractKernelTestCase
+ */
+class AbstractKernelTestCase extends KernelTestCase
+{
+    use TestCleanUpTearDownTrait;
+}
```
Comment (nicolasThal) [inline BaseBundle/Tests/AbstractTest/AbstractKernelTestCase.php]: eof

File: BaseBundle/Tests/AbstractTest/AbstractWebTestCase.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+/**
+ * Class AbstractWebTestCase
+ */
+class AbstractWebTestCase extends WebTestCase
+{
+    use TestCleanUpTearDownTrait;
+}
```
Comment (nicolasThal) [inline BaseBundle/Tests/AbstractTest/AbstractWebTestCase.php]: eof

File: BaseBundle/Tests/AbstractTest/TestCleanUpTearDownTrait.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        $refl = new ReflectionObject($this);
+        foreach ($refl->getProperties() as $prop) {
+            if (!$prop->isStatic() && 0 !== strpos($prop->getDeclaringClass()->getName(), 'PHPUnit_')) {
+                $prop->setAccessible(true);
+                $prop->setValue($this, null);
+            }
+        }
+    }
+}
```
Comment (nicolasThal) [inline BaseBundle/Tests/AbstractTest/TestCleanUpTearDownTrait.php]: eof

File: BaseBundle/Tests/AbstractTest/AbstractKernelTestCase.php (hunk 2 above)
Comment (nicolasThal) [inline BaseBundle/Tests/AbstractTest/AbstractKernelTestCase.php]: You should take care of the kernelTestCase, the tearDown method ensure that the kernel is shutdown

**Comment by itkg-alavieille** [comment]
@nicolasThal

File: BaseBundle/Tests/AbstractTest/AbstractKernelTestCase.php [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+namespace OpenOrchestra\BaseBundle\Tests\AbstractTest;
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+/**
+ * Class AbstractKernelTestCase
+ */
+class AbstractKernelTestCase extends KernelTestCase
+{
+    use TestCleanUpTearDownTrait;
```
Comment (nicolasThal) [inline BaseBundle/Tests/AbstractTest/AbstractKernelTestCase.php]: You should take care of the kernelTestCase, the tearDown method ensure that the kernel is shutdown

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**